src/templates/language_selector.html

```diff
@@ -1,4 +1,4 @@
-<ul>
+<div id="language-selector">
```

Comment:
was wondering why this markup is getting removed & re-added at all. that explains that.